db/silkworm/db/stages.hpp

```diff
@@ -27,26 +27,51 @@ List of stages keys stored into SSP2 table
 
 namespace silkworm::db::stages {
 
-constexpr const char* kAccountHistoryKey{"AccountHistoryIndex"};
-constexpr const char* kBlockHashesKey{"BlockHashes"};
-constexpr const char* kBlockBodiesKey{"Bodies"};
-constexpr const char* kExecutionKey{"Execution"};
-constexpr const char* kFinishKey{"Finish"};
-constexpr const char* kHashStateKey{"HashState"};
-constexpr const char* kHeadersKey{"Headers"};
-constexpr const char* kIntermediateHashesKey{"IntermediateHashes"};
-constexpr const char* kLogIndexKey{"LogIndex"};
-constexpr const char* kSendersKey{"Senders"};
-constexpr const char* kStorageHistoryIndexKey{"StorageHistoryIndex"};
-constexpr const char* kTxLookupKey{"TxLookup"};
-constexpr const char* kTxPoolKey{"TxPool"};
+// clang-format off
+
+constexpr const char* kHeadersKey{"Headers"};                         // Headers are downloaded, their Proof-Of-Work validity and chaining is verified
+constexpr const char* kBlockHashesKey{"BlockHashes"};                 // Headers Number are written, fills blockHash => number bucket
+constexpr const char* kBlockBodiesKey{"Bodies"};                      // Block bodies are downloaded, TxHash and UncleHash are getting verified
+constexpr const char* kSendersKey{"Senders"};                         // "From" recovered from signatures
+constexpr const char* kExecutionKey{"Execution"};                     // Executing each block w/o buildinf a trie
+constexpr const char* kIntermediateHashesKey{"IntermediateHashes"};   // Generate intermediate hashes, calculate the state root hash
+constexpr const char* kHashStateKey{"HashState"};                     // Apply Keccak256 to all the keys in the state
+constexpr const char* kAccountHistoryKey{"AccountHistoryIndex"};      // Generating history index for accounts
+constexpr const char* kStorageHistoryIndexKey{"StorageHistoryIndex"}; // Generating history index for storage
+constexpr const char* kLogIndexKey{"LogIndex"};                       // Generating logs index (from receipts)
+constexpr const char* kCallTracesKey{"CallTraces"};                   // Generating call traces index
+constexpr const char* kTxLookupKey{"TxLookup"};                       // Generating transactions lookup index
+constexpr const char* kTxPoolKey{"TxPool"};                           // Starts Backend
+constexpr const char* kFinishKey{"Finish"};                           // Nominal stage after all other stages
+
+constexpr const char* kMiningCreateBlockKey{"MiningCreateBlock"};     // Create block for mining
+constexpr const char* kMiningExecutionKey{"MiningExecution"};         // Execute mining
+constexpr const char* kMiningFinishKey{"MiningFinish"};               // Mining completed
+
+// clang-format on
+
+constexpr const char* AllStages[]{
```

Comment:
Better name kAllStages for consistency with [Google's Style Guide](https://google.github.io/styleguide/cppguide.html#Constant_Names).